sympy/physics/continuum_mechanics/beam.py

```diff
@@ -332,13 +345,29 @@ def boundary_conditions(self):
         >>> b.bc_deflection = [(0, 2)]
         >>> b.bc_slope = [(0, 1)]
         >>> b.boundary_conditions
-        {'deflection': [(0, 2)], 'slope': [(0, 1)]}
+        {'bending moment': [], 'deflection': [(0, 2)], 'shear force': [], 'slope': [(0, 1)]}
```

Comment:
```suggestion
        {'bending_moment': [], 'deflection': [(0, 2)], 'shear_force': [], 'slope': [(0, 1)]}
```
I think making these keys valid Python identifiers may be best, for unknown future use (and tab completion in IDEs).

Reply:
also matches the property names